packages/server/graphql/mutations/updateGitLabDimensionField.ts

```diff
@@ -0,0 +1,77 @@
+import {GraphQLID, GraphQLNonNull, GraphQLString} from 'graphql'
+import {SubscriptionChannel} from 'parabol-client/types/constEnums'
+import MeetingPoker from '../../database/types/MeetingPoker'
+import upsertGitLabDimensionFieldMap from '../../postgres/queries/upsertGitLabDimensionFieldMap'
+import {isTeamMember} from '../../utils/authorization'
+import publish from '../../utils/publish'
+import {GQLContext} from '../graphql'
+import UpdateGitLabDimensionFieldPayload from '../types/UpdateGitLabDimensionFieldPayload'
+
+interface Args {
```

Comment:
-1 could you write this as a resolver using the new SDL pattern? that way these args will be provided for you

Reply:
Nice, it was cool to see the new pattern! The type safety is so good 🙌  Update pushed!